core/src/components/radio/radio.md.vars.scss

```diff
@@ -11,7 +11,7 @@ $radio-md-color-on:                  current-color(base) !default;
 $radio-md-background-color-focused:  ion-color(primary, tint) !default;
 
 /// @prop - Color of the unchecked radio
-$radio-md-color-off:                 $text-color-step-600 !default;
+$radio-md-color-off:                 rgb($text-color-rgb, 0.60) !default;
```

Comment:
Why did we use an opacity rather than a different step color value? This will potentially change how a radio color mixes with the underlying element since it has an opacity now.

Reply:
I actually made it use a step color at first, but then I took a look at how the [Material Design 2 kit](https://www.figma.com/community/file/778763161265841481) does it and they use opacity for both checkbox and radio:

<img width="1222" alt="checkbox-md2" src="https://github.com/ionic-team/ionic-framework/assets/6577830/3c07aa1e-8487-4039-b7fe-52d5a1789c8f">
<img width="1221" alt="radio-md2" src="https://github.com/ionic-team/ionic-framework/assets/6577830/ccfb8eab-5a36-4507-a502-ffc700d9623b">

Reply:
Thanks! I see https://material-components.github.io/material-components-web-catalog/#/component/radio also does the same thing.